brave/brave-core
# Add shields settings

Resolves https://github.com/brave/brave-browser/issues/16300
Resolves https://github.com/brave/brave-browser/issues/16299

## Submitter Checklist:

- [x ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Check ad blocker, fingerprinting, cookies settings as the code has been refactored
2. Check there is no conflict between shields settings and the global settings
3. Check that shields settings are working on the loaded site after swiping out and re-starting the app
4. Check for update too, as we had that issue before
